database/sqlconfig.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  user: 'root',
```

Comment:
You will likely need to add a host to get your proxy running and in general it is good practice to have it listed regardless.